### PR TITLE
Create CI Deploy infra

### LINKED ISSUE #4: Create Deploy Github actions for flow-archive-access
There is currently no deploy process in place, we need to have the same deploy route for flow-archive-access as flow-archive

### PR DESCRIPTION
**Pull request by koko1123**
## Goal of this PR
Creates Deploy files and dev convenience tools 

## Checklist

- [x] Is on the right branch
- [x] Documentation is up-to-date
- [x] Tests are up-to-date

### CODE REVIEW

**Review by sjonpaulbrown — APPROVED** [review]

File: api/server.go [hunk 1]
```diff
@@ -547,7 +546,7 @@ func (s *Server) GetEventsForHeightRange(_ context.Context, in *access.GetEvents
 			return nil, fmt.Errorf("could not get header at height %d: %w", height, err)
 		}
 
-		timestamp, err := ptypes.TimestampProto(header.Timestamp)
+		timestamp := timestamppb.New(header.Timestamp)
 		if err != nil {
```
Comment (peterargue) [inline api/server.go]: do we still need to check this error?

File: api/server.go [hunk 2]
```diff
@@ -601,7 +600,7 @@ func (s *Server) GetEventsForBlockIDs(_ context.Context, in *access.GetEventsFor
...
```
Comment (peterargue) [inline api/server.go]: do we still need to check this error?

File: api/server.go [hunk 3]
```diff
@@ -397,7 +396,7 @@ func (s *Server) GetTransactionsByBlockID(ctx context.Context, in *access.GetTra
 	}
 
 	chain := header.ChainID.Chain()
-	systemTx, err := blueprints.SystemChunkTransaction(chain)
+	systemTx, _ := blueprints.SystemChunkTransaction(chain)
```
Comment (peterargue) [inline api/server.go]: why are we ignoring this error?

File: api/server.go (hunk 2 above)
Comment (koko1123) [reply api/server.go]: right, removed the check. The IDE linter made these fixes

File: api/server.go [hunk 4]
```diff
@@ -398,6 +397,9 @@ func (s *Server) GetTransactionsByBlockID(ctx context.Context, in *access.GetTra
 
 	chain := header.ChainID.Chain()
 	systemTx, err := blueprints.SystemChunkTransaction(chain)
+	if err != nil {
+		return nil, fmt.Errorf("could not get system transactipn for height %x: %w", height, err)
```
Comment (peterargue) [inline api/server.go]:
```suggestion
		return nil, fmt.Errorf("could not get system transaction for height %x: %w", height, err)
```

**Review by peterargue — APPROVED** [review]

**Pull request merged**